fdbserver/include/fdbserver/DDTxnProcessor.h

```diff
@@ -22,7 +22,6 @@
 #define FOUNDATIONDB_DDTXNPROCESSOR_H
 
 #include "fdbserver/Knobs.h"
-#include "fdbserver/DataDistribution.actor.h"
 #include "fdbserver/MoveKeys.actor.h"
```

Comment:
There is still a `.actor.h` file included here.

Reply:
I thought I removed it in [b48b1e93f](https://github.com/apple/foundationdb/pull/7557/commits/b48b1e93f97ed1a06e4c806d4b609834ba34e6a1). There was a git merge problem... Lol

Reply:
IMO it's okay to include a `.actor.h` header because the actor header `fdbserver/MoveKeys.actor.h` already has `actorcompiler.h`. The case need `ctorcompiler.h` explicitly is when the header file itself has ACTOR.